plugin/storage/es/spanstore/reader.go

```diff
@@ -109,7 +106,7 @@ type SpanReaderParams struct {
 	SpanIndex           cfg.IndexOptions
 	ServiceIndex        cfg.IndexOptions
 	TagDotReplacement   string
-	Archive             bool
+	ReadAlias           string
```

Comment:
```suggestion
	ReadAliasSuffix           string
```
let's be more precise, this code already has unnecessarily high complexity to complicate it further with inaccurate names